pom.xml

```diff
@@ -1036,29 +1036,6 @@
           </dependency>
         </dependencies>
       </plugin>
-      <!-- jacoco code coverage testing library -->
```

Comment:
Please reinstate this section of the POM. It is useful if you want to run Jacoco for analyzing code coverage during testing. You could add a profile that enables Jacoco as well.